runtime/v1/linux/proc/exec.go

```diff
@@ -142,10 +144,11 @@ func (e *execProcess) Kill(ctx context.Context, sig uint32, _ bool) error {
 
 func (e *execProcess) kill(ctx context.Context, sig uint32, _ bool) error {
 	pid := e.pid.get()
-	if pid != 0 {
-		if err := unix.Kill(pid, syscall.Signal(sig)); err != nil {
-			return errors.Wrapf(checkKillError(err), "exec kill error")
-		}
+	if pid <= 0 {
```

Comment:
is it possible to kill a creating exec process(pid=0) here? It seems we do create-and-run exec process in one transaction in client side. If no, the change looks good to me.

Reply:
updated to make this correct